executor/explain_test.go

```diff
@@ -155,6 +156,19 @@ func (s *testSuite1) checkMemoryInfo(c *C, tk *testkit.TestKit, sql string) {
 	}
 }
 
+func (s *testSuite1) TestMemoryUsageAfterClose(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t (v int, k int, key(k))")
+	tk.MustExec("insert into t values (1, 1), (1, 1), (1, 1), (1, 1), (1, 1)")
```

Comment:
I guess these values only take 1 chunk? How about set `tidb_max_chunk_size` to 2 (or 1) so that there is some parallelism during the test?

Reply:
Updated.
I insert more rows(MaxChunkSize * 5) into this table now.